### PR TITLE
Handle user and channel when converting a reference to a path

### PR DESCRIPTION
**Pull request by anders-zpt**

### CODE REVIEW

**Review by memsharded — APPROVED** [review]
Thanks very much for this fix!

**Comment by memsharded** [comment]
Could you please check and sign the CLA? Many thanks

File: extensions/commands/art/cmd_property.py [hunk 1]
```diff
@@ -37,10 +37,12 @@ def _get_path_from_ref(ref):
         recipe_ref = RecipeReference.loads(ref)
         package_ref = None
 
-    rrev_path = f"/_/{recipe_ref.revision}" if recipe_ref.revision else ""
+    user = recipe_ref.user if recipe_ref.user else "_"
+    channel = recipe_ref.channel if recipe_ref.channel else "_"
+    rrev_path = f"/{channel}/{recipe_ref.revision}" if recipe_ref.revision else ""
```
Comment (danimtb) [inline extensions/commands/art/cmd_property.py:42]: Better to leave rrev_path mostly as it is and use directly the channel value in the final path formatting
```suggestion
    rrev_path = f"/{recipe_ref.revision}" if recipe_ref.revision else ""
```

File: extensions/commands/art/cmd_property.py [hunk 2]
```diff
@@ -37,10 +37,12 @@ def _get_path_from_ref(ref):
...
     pkgid_path = f"/package/{package_ref.package_id}" if package_ref and package_ref.package_id else ""
     prev_path = f"/{package_ref.revision}" if package_ref and package_ref.revision else ""
-    return f"_/{recipe_ref.name}/{recipe_ref.version}{rrev_path}{pkgid_path}{prev_path}"
+    return f"{user}/{recipe_ref.name}/{recipe_ref.version}{rrev_path}{pkgid_path}{prev_path}"
```
Comment (danimtb) [inline extensions/commands/art/cmd_property.py:45]: As described in the previous comment:
```suggestion
    return f"{user}/{recipe_ref.name}/{recipe_ref.version}/{channel}{rrev_path}{pkgid_path}{prev_path}"
```

**Review by danimtb — COMMENTED** [review]
Thanks for the contribution! Indeed, user and channel fields were not taken into account. I have added a minor code suggestion 😄

**Comment by danimtb** [comment]
Awesome! Thanks @anders-zpt

**Pull request merged**